app/model/jwt_user.py

```diff
@@ -3,7 +3,7 @@
 from pydantic import BaseModel
 
 
-class JWTUser(BaseModel):
+class UserContext(BaseModel):
```

Comment:
Do we need to change the name of the module from `jwt_user.py` as well?

Reply:
I've renamed to 'user' :)